types/responsepolicy.pp

```diff
@@ -0,0 +1,12 @@
+type Dns::ResponsePolicy = Array[
+  Struct[{
+    zone           => String[1],
```

Comment:
I imagine `zone` is a domain, so we can enforce a stricter type
```suggestion
    zone           => Stdlib::Fqdn,
```

Reply:
It seems `Stdlib::Fqdn` might be too restrictive for the zone field
example: rpz
Using `String[1]` will allow more flexibility in specifying zone to allow both FQDN and non-FQDN values

Reply:
Technically `Stdlib::Fqdn` allows `rpz`. It may be poorly named, but I'd prefer that. Strong validation is important to me.

Reply:
yes, you are correct.